spiffetls/tests/check_peerid.c

```diff
@@ -0,0 +1,95 @@
+#include "spiffetls/src/dial.h"
+#include "spiffetls/src/mode.h"
+#include "spiffetls/src/peerid.h"
+#include "spiffetls/tlsconfig/src/config.h"
+#include <check.h>
+#include <stdio.h>
+#include <unistd.h>
+
+START_TEST(test_spiffetls_PeerIDFromConn)
+{
+    /* Hot path */
+    system("./tls_server resources/good-leaf-only.pem "
```

Comment:
Remember other tests should run in parallel and not fail.
This fails to acquire the port if another "tls_server" is also running, including for other tests.
Changing the port used should suffice, or even better a randomly chosen passed from the test to the "tls_server" program via args.